source/FpsConfig.h

```diff
@@ -85,7 +85,11 @@ class PlayerConfig {
 	float			movementRestrictionZ = 5.0f;				///< Player's restricted movement span along Z
 	bool			restrictedMovementEnabled = false;			///< If true, player's movement will be restricted along X and Z
 	bool			counterStrafing = false;					///< If true, counter strafing will be enabled. (defaults to false.)
-
+	bool			propagatePlayerConfigsToAll = false;		///< If true, all players config from the server will be broadcast to clients
+	bool			propagatePlayerConfigsToSelectedClient = false; ///< If true, all players config from the server will be sent to selected client
+	bool			readFromFile = false;						///< Read client configs from file or not
+	String			selectedClient = "Client 1";				///< Indicates the client that player configs will be propagated to
+	Array <PlayerConfig>  clientPlayerConfigs;
```

Comment:
It's a bit confusing (and potentially dangerous to memory management) to have an `Array<Object>` be defined as a member inside of that object... Like above this feels like it should be in a server exclusive config file.

Reply:
I'll also keep this to be implemented in the next iteration.